code/ui/components/src/new/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { forwardRef } from 'react';
+import { styled } from '@storybook/theming';
+import TextareaAutoResize from 'react-textarea-autosize';
+
+interface TextareaProps {
+  disabled?: boolean;
+  placeholder?: string;
+  value?: string;
+}
+
+export const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(({ ...props }, ref) => {
+  return <StyledTextarea ref={ref} {...props} />;
+});
+
+Textarea.displayName = 'Textarea';
```

Comment:
Is this needed? Why?

Reply:
Because it's a forwardRef with an arrow function inside.

Reply:
It is needed to make sure React devtools function correctly. This triggers an error otherwise.